website/docs/d/nsxt_app_port_profile.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_app_port_profile"
+sidebar_current: "docs-vcd-data-source-nsxt-app-port-profile"
+description: |-
+  Provides a data source to read NSX-T Application Port Profiles. Application Port Profiles include 
+  a combination of a protocol, and a port, or a group of ports, that is used for firewall and NAT
+  services on the edge gateway. In addition to the default port profiles that are preconfigured for
+  NSX-T Data Center.
+---
+
+# vcd\_nsxt\_app\_port\_profile
+
+Supported in provider *v3.3+* and VCD 10.1+ with NSX-T backed VDCs.
+
+Provides a data source to read NSX-T Application Port Profiles. Application Port Profiles include a
+combination of a protocol and a port, or a group of ports, that is used for firewall and NAT
+services on the edge gateway. In addition to the default port profiles that are preconfigured for
```

Comment:
Just following the capitalization style:
```suggestion
services on the Edge Gateway. In addition to the default port profiles that are preconfigured for
```

Reply:
True. Fixed.